incubator/java-openliberty/image/project/Dockerfile

```diff
@@ -12,23 +12,29 @@ RUN cd /project && mvn -B install dependency:go-offline -DskipTests
 COPY ./util /project/util
 RUN  /project/util/check_version build
 
-# Copy the user's application pom and resolve all dependencies, also trigger Open Liberty install
+# Download Openliberty here to prevent a re-download
+RUN cd /project && mvn -B liberty:install-server liberty:create
+
+# Copy the user's application pom and resolve all dependencies
 COPY ./user-app/pom.xml /project/user-app/pom.xml
+RUN cd /project/user-app && mvn package -Dskip=true -DskipTests
+
+# validate the config.
 COPY ./validate.sh /project/user-app/validate.sh
 RUN cd /project/user-app && ./validate.sh
 
 # Copy and build the application source, and run unit tests
 COPY ./user-app/src /project/user-app/src
-RUN cd /project/user-app && mvn -B liberty:create package
+RUN cd /project/user-app && mvn package -DskipTests
```

Comment:
Since we no longer have a LMP parent (like we do in java-microprofile), I don't think `package` is enough (no goals are bound to it). We probably want `package liberty:install-feature liberty:deploy`